docs/python_armada_client.md

```diff
@@ -11,6 +11,9 @@ title: armada_client package
 
 Armada Python GRPC Client
 
+For the api definitions:
+[https://github.com/G-Research/armada/blob/master/docs/api.md](https://github.com/G-Research/armada/blob/master/docs/api.md)
```

Comment:
this should instead link to armadaproject.io/api -- we have tended to prioritize making the website links better than the github-based links